submit.go

```diff
@@ -224,68 +257,86 @@ func ParsArgs() *JobSubmit {
 	return jobSubmit
 }
 
-// GetStatus get status of the job from the API
-func (c *Client) GetStatus(projectID string, jobID string) *JobStatus {
-
-	job := &JobStatus{}
-	path := fmt.Sprintf(DataProcessingStatus, url.QueryEscape(projectID), url.QueryEscape(jobID))
-	if err := c.OVH.Get(path, job); err != nil {
-		log.Fatalf("Unable to retrieve status for job: %s", err)
+//poll Status
+func Loop(c *Client, job *JobStatus) {
+	sigs := make(chan os.Signal, 2)
+	signal.Notify(sigs, os.Interrupt, syscall.SIGTERM)
+	var err error
+statusLoop:
+	for {
+		select {
+		case <-sigs:
+			var s string
+			fmt.Printf("Do you want to kill the Job (y/N): ")
```

Comment:
why y/N and not y/n or Y/N?

Reply:
y/N convention that tell us N is default

Reply:
Knew that, but completely forgot, sorry ^^'